addons/sale/models/sale_order.py

```diff
@@ -1295,6 +1295,20 @@ def _get_invoiceable_lines(self, final=False):
 
         return self.env['sale.order.line'].browse(invoiceable_line_ids + down_payment_line_ids)
 
+    def process_zero_value_orders(self):
```

Comment:
This should in `website_sale` module also it should be private and it could in `main.py` with order as an arg